src/store/types.ts

```diff
@@ -97,6 +97,7 @@ export interface AudioDetail extends BaseMediaDetail<'audio'> {
   sample_rate?: number
   alt_files?: any
   filetype?: string
+  peaks?: any
```

Comment:
We can actually set this to `number[]` specifically :slightly_smiling_face: 

```suggestion
  peaks?: number[]
```

It might also be always defined, in which case we could remove the `?` but I'm not confident about that... and it's better safe than sorry when working with API responses anyway.

Reply:
@ChrisCoastal just wanted to make sure you saw this review. Thanks for your contribution so far!

Reply:
Hello @sarayourfriend and @zackkrida and thank you for the feedback and welcome to Openverse! I am very thankful for the guidance and opportunity. Apologies, as I've just seen this followup now, so I will changes and submit. Thanks!